### ISSUE TITLE
Not work on windows 7 32 bit

### ISSUE THREAD

**Issue by Polish-Lapille** [Not-My-Problem/Console-Calculator#2]
Doesn't work after downloading net 6.0. When turned on, the application turns off.

**Comment by Not-My-Problem** [comment]
Thank you for the information. We will try to fix it as soon as possible.

**Comment by Not-My-Problem** [comment]
Have you tried running it in windows 7 compatibility mode?

**Comment by Not-My-Problem** [comment]
Masz rację, nie działa na maszynie wirtualnej. Próbuję to teraz naprawić

**Comment by Polish-Lapille** [comment]
You are right it doesn't work on virtual machine. I'm trying to fix this right now

**Comment by Not-My-Problem** [comment]
This will be fixed in the next update

**Comment by Kamiieee** [comment]
It will be fixed tommorow, the owner needs to go now.

**Comment by Not-My-Problem** [comment]
Anyone else here? I am changing the language from C# to C++. It is possible that the first version will be today. I'll try to translate tomorrow.
